Libraries/Components/TextInput/TextInput.js

```diff
@@ -152,6 +152,10 @@ const DataDetectorTypes = [
  * );
  * ```
  *
+ * By default, when you assign a `height` style to TextInput, the text & placeholder
```

Comment:
That's not quite true. At least on iOS it depends on 'multiline' property. Right?
I rather prefer to elaborate the **desired** behaviour (for iOS/Android and multiline/singleline) and fix the current implementation. And probably mentioned here current limitations and bugs.